krab/utils.go

```diff
@@ -0,0 +1,33 @@
+package krab
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
hmmm not a big fan of this package, I find `fmt.Errorf("something: %s", err.Error())` more than sufficient. what are you thoughts?

Reply:
How come? is it due to the size, or..?, I ideally want to maintain as much compatibility with how ipfs handles keystores, which is where I copied this code from

Reply:
well it just doesn't do anything in most cases - I don't see a situation where you'd really want to invoke `Cause()`, since our error checking isn't very granular anyways. I think we can use the standard error interface without any issue